src/application/look-up-wallet-address/schema.ts

```diff
@@ -0,0 +1,5 @@
+import { z } from 'zod';
+
+export const twitterIdSchema = z.object({
+  twitterIDs: z.record(z.string(), z.string()),
```

Comment:
I am not sure if this will always have value of a string, can you double check by trying to call this api with some non-existing username?
I remember one of our api's return empty string and other returns null or smth like that.

Reply:
in this case it just returns an empty object

```twitterIDs: {}```